client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -168,6 +168,16 @@ class LoggedInForm extends Component {
 		}
 	}
 
+	isFromJPO() {
```

Comment:
Just a note as we'll be working on this in the future. Lots of similar logic between `isWoo`, `isFromJPO` and `isSSO`. We might consider consolidating it:

https://github.com/Automattic/wp-calypso/blob/58d602e61c72ea39235f324a175f39923752c1de/client/jetpack-connect/authorize-form.jsx#L82-L99

I think this is OK for this PR.